test/parallel/test-blob.js

```diff
@@ -220,3 +220,18 @@ assert.throws(() => new Blob({}), {
     });
   });
 }
+
```

Comment:
```suggestion

// Based on test/fixtures/wpt/FileAPI/blob/Blob-constructor.any.js
// which is disabled due to dependency on other web APIs.
// Exercises  #40705
```

@aduh95 does this look good ?